single.php

```diff
@@ -34,13 +34,13 @@
 									endif;
 
 									if(get_the_content()):
-										echo "<p>".get_the_content()."</p>";
+											the_content();
```

Comment:
@Huyeng Shouldn't we just call `the_content()` function as we do with `odm_echo_extras();` and `echo_downloaded_documents();`